### PR TITLE
Removed class variable

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: lib/fastlane_core/itunes_transporter.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def execute_transporter(command)
       @errors = []
       @warnings = []
 
-      if defined?@@hide_transporter_output
```
Comment (i2amsam) [inline lib/fastlane_core/itunes_transporter.rb]: Space after `defined?`

**Comment by i2amsam** [comment]
This looks good, but one side of each build failed, restarted them.  It'd be good to cleanup the hound issue too.

**Comment by KrauseFx** [comment]
Code style verification works fine for me locally, waiting for Travis now

**Comment by KrauseFx** [comment]
Works now :+1:

**Pull request merged**